docs/snapcraft-syntax.md

```diff
@@ -20,24 +20,33 @@ contain.
  * `config` (string)  
    Path to the runnable in snap that will be used as the [Snappy config
    interface](https://developer.ubuntu.com/snappy/guides/config-command/).
- * `services` (yaml subsection)  
-   A set of keys representing service names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
- * `binaries` (yaml subsection)  
-   A set of keys representing binary names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
+ * `apps` (yaml subsection)  
+   A map of keys for integration points for the system. Either as a daemon
+   or command line accessible binary.
+   * `command` (string)
+     Specifies the internal command to expose.
+   * `daemon` (string)
+     If present, integrates the runnable as a system service. Valid values are
+     forking and simple.
+   * `stop-command` (string)
+     Requires `daemon` to be specified and represents the command to run to
+     effectively stop the service.
+   * `stop-timeout` (integer)
+     Requires `daemon` to be specified. It is number in seconds to indicate how
```

Comment:
It is number of seconds that the system will wait for the service to stop.
I'm not sure if this sounds better, so @kyrofa please.